CODE_OF_CONDUCT.md

```diff
@@ -67,8 +67,8 @@ members of the project's leadership.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 2.1,
```

Comment:
Just the link is updated the file ain't 😅

Reply:
Okay I will do it. Thanks.